trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/VLM.py

```diff
@@ -438,20 +440,22 @@ def VLM(conditions,settings,geometry):
     TFZ[SPC<0] = np.abs(XCOS)[SPC<0]*np.sign(DCP_LE)[SPC<0]
 
     CAXL = CAXL - TFX*CSUC
-
+    
     # Add a dimension into the suction to be chordwise
     CNC   = CNC + CSUC*np.sqrt(1+T2)*TFZ
-
+    
     # FCOS AND FSIN ARE THE COSINE AND SINE OF THE ANGLE BETWEEN
     # THE CHORDLINE OF THE IR-STRIP AND THE X-AXIS    
     FCOS = np.cos(ZETA)
     FSIN = np.sin(ZETA)
+    
 
     # BFX, BFY, AND BFZ ARE THE COMPONENTS ALONG THE BODY AXES
     # OF THE STRIP FORCE CONTRIBUTION.
     BFX = -  CNC *FSIN + CAXL *FCOS
     BFY = - (CNC *FCOS + CAXL *FSIN) *SID
     BFZ =   (CNC *FCOS + CAXL *FSIN) *COD
+    
```

Comment:
Stop adding random spaces. This script is long enough. One blank line with double comment lines is sufficient.

Reply:
this must have been an issue with merging with develop since I did not touch this. removed extra spaces.